### PR TITLE
Controllers and actions

### PR DESCRIPTION
**Pull request by muhammedb8826**
### In this milestone I've worked on the following

- Created a controllers and their actions
- Created a views for each actions
- Added resources :users inside routes.rb file

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
+  resources :users
```
Comment (prolajumokeoni) [inline config/routes.rb]: - [ ] Kindly setup routes for only users and posts index and show routes, your routes config should look like the example below ⬇️ 

 ```
 resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @muhammedb8826 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  User Controller, and   Views setup correctly 👏
  




## Required Changes ♻️

- [ ] Kindly add `posts_controller` with `#index` and `#show` actions, It's okay for the actions to be empty for now, please add corresponding view files for each of the posts  controller actions 👍🏿 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (muhammedb8826) [reply config/routes.rb]: Thank you for the feedback, I will do that.

**Review by Olamarx — APPROVED** [review]
Hi @muhammedb8826, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Requested changes fixed ✔️
- Post controller ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by muhammedb8826** [comment]
Thank you for approving my project, I appreciated!

**Pull request merged**